web3/utils/signing.py

```diff
@@ -9,3 +16,38 @@ def signature_wrapper(message, version=b'E'):
         return preamble + size + message
     else:
         raise NotImplementedError("Only the 'Ethereum Signed Message' preamble is supported")
+
+
+def annotate_transaction_with_chain_id(transaction_parts, raw_v):
+    '''
+    Extends transaction with chain ID, according to EIP-155
+    See details at https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+    @return (transaction_parts, chain_id, v)
+    '''
+    assert len(transaction_parts) == 6
+    (chain_id, v) = extract_chain_id(raw_v)
+    if chain_id is None:
+        chain_id_extension = []
+    else:
+        chain_id_extension = [to_bytes(chain_id), b'', b'']
+    return (transaction_parts + chain_id_extension, chain_id, v)
+
+
+def extract_chain_id(raw_v):
+    '''
+    Extracts chain ID, according to EIP-155
+    @return (chain_id, v)
+    '''
+    above_id_offset = raw_v - CHAIN_ID_OFFSET
+    if above_id_offset < 0:
+        return (None, raw_v)
+    else:
+        (chain_id, v_bit) = divmod(above_id_offset, 2)
+        return (chain_id, v_bit + V_OFFSET)
+
+
+def sign_transaction_hash(account, transaction_hash, chain_id):
+    signature = account.sign_msg_hash(transaction_hash)
+    (v_raw, r, s) = (getattr(signature, part) for part in 'vrs')
```

Comment:
I believe you can just access `signature.vrs` making this:

```python
(v_raw, r, s) = signature.vrs
```

Reply:
:man_facepalming: thanks, I missed that attribute